raiden/transfer/mediation_fee.py

```diff
@@ -74,3 +77,40 @@ def reversed(self: T) -> T:
         )
         self._update_penalty_func()
         return reversed_instance
+
+
+def linspace(start: TokenAmount, stop: TokenAmount, num: int) -> List[TokenAmount]:
+    """ Returns a list of num numbers from start to stop (inclusive). """
+    assert num > 1
+    assert start <= stop
+
+    step = (stop - start) / (num - 1)
+
+    result = []
+    for i in range(num):
+        result.append(TokenAmount(start + round(i * step)))
+
+    return result
+
+
+def calculate_rebalancing_fees(
+    our_balance: Balance, partner_balance: Balance
+) -> List[Tuple[TokenAmount, FeeAmount]]:
+    """ Calculates a quadratic rebalancing curve.
+
+    The penalty term takes the value `UINT256_MAX` at the extrema.
+    """
+    total_balance = TokenAmount(our_balance + partner_balance)
```

Comment:
Do we have a clear rule when to use "capacity" and when to use "balance"?

Reply:
I think here balance is right, as balance is the available "bandwidth" for the channel.
```
    return Balance(
        sender.contract_balance
        - sender.total_withdraw
        - sender_transferred_amount
        + receiver_transferred_amount
    )
```

Capacity would be this balance, minus the locked amount.

Reply:
Then we should change the naming in `FeeSchedule` from `Capacity` to `Balance`, since we exclude locked amounts there, too.